index.html

```diff
@@ -39,111 +39,113 @@
+                    <div class="feature__img">
                         <img src="./images/responsive.svg" alt="">
                     </div>
-                    <h4 class="card__title">fully responsive</h4>
-                    <p class="card__info">Looks amazing on any device: smartphone, tablet, laptop and desktop.</p>
-                </div>
-                <div class="card">
-                    <div class="card__img">
+                    <h4 class="feature__title">fully responsive</h4>
+                    <p class="feature__info">Looks amazing on any device: smartphone, tablet, laptop and desktop.</p>
+                </li>
+                <li class="feature">
+                    <div class="feature__img">
                         <img src="./images/customizable.svg" alt="">
                     </div>
-                    <h4 class="card__title">customizable</h4>
-                    <p class="card__info">Change the colors, pictures or any of the section to suit you needs.</p>
-                </div>
-                <div class="card">
-                    <div class="card__img">
+                    <h4 class="feature__title">customizable</h4>
+                    <p class="feature__info">Change the colors, pictures or any of the section to suit you needs.</p>
+                </li>
+                <li class="feature">
+                    <div class="feature__img">
                         <img src="./images/design.svg" alt="">
                     </div>
-                    <h4 class="card__title">slick and beautiful design</h4>
-                    <p class="card__text">Trendy and fresh design, fits any website.</p>
-                </div>
-            </div>
+                    <h4 class="feature__title">slick and beautiful design</h4>
+                    <p class="feature__text">Trendy and fresh design, fits any website.</p>
+                </li>
+            </ul>
         </div>
     </section>
 
-    <section class="partner">
+    <aside class="partner">
         <div class="partner__container container">
             <img class="partner__logos" src="./images/logos.png" alt="partner logos">
         </div>
-    </section>
+    </aside>
 
-    <section class="description">
+    <article class="description">
         <div class="description__container">
             <img class="description__img" src="./images/imac.png" alt="imac">
-            <div class="description__info">
+            <article class="description__info">
                 <h2 class="description__title">simple and beautiful</h2>
                 <p class="description__text">Use nova theme for your next web project. It is completely customizable so
                     you can change any of the sections to fit your needs. Nova Theme is Free for any kind of use,
                     personal and commercial. Have fun and good luck!</p>
-            </div>
+            </article>
         </div>
-    </section>
+    </article>
```

Comment:
W tym przypadku 2 x `article` jest nie potrzebne. Jak to wyjaśnisz? Raczej zostałbym `section > article`